lib/osf-components/addon/components/osf-dialog/component.ts

```diff
@@ -30,6 +30,12 @@ export default class OsfDialog extends Component {
 
     @or('isOpen', 'hasTriggeredOpen') shouldBeOpen!: boolean;
 
+    @computed('renderInPlace')
+    get _renderInPlace() {
+        return this.renderInPlace || typeof document === 'undefined' ||
```

Comment:
When is `typeof document === 'undefined'` ever true?

Reply:
this was for safety. to avoid typeError, 'getElementId' of undefined.